src/Tasks/Common/src/ConflictResolution/ResolvePackageFileConflicts.cs

```diff
@@ -76,6 +79,15 @@ protected override void ExecuteCore()
             var platformConflictScope = new ConflictResolver<ConflictItem>(packageRanks, log);
             var platformItems = PlatformManifests?.SelectMany(pm => PlatformManifestReader.LoadConflictItems(pm.ItemSpec, log)) ?? Enumerable.Empty<ConflictItem>();
 
+            //  Also treat assemblies from FrameworkList.xml as platform assemblies
+            if (TargetFrameworkDirectories != null && TargetFrameworkDirectories.Any())
+            {
+                platformItems = platformItems.Concat(TargetFrameworkDirectories.SelectMany(tfd =>
```

Comment:
Consider not using `SelectMany` + `Concat` if this is a hot path where allocations matter.